src/main/java/org/spburegistry/backend/entity/Project.java

```diff
@@ -0,0 +1,76 @@
+package org.spburegistry.backend.entity;
+
+import jakarta.annotation.Nullable;
+import jakarta.persistence.*;
+import lombok.*;
+
+import java.util.Set;
+
+@EqualsAndHashCode(callSuper = false)
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@Table(name="projects")
+@Data
+public class Project extends BaseEntity {
+
+    @Column(nullable = false)
+    private String name;
+
+    @Column(nullable = false)
+    private String description;
+
+    @ManyToOne
+    @JoinColumn(name = "studying_program_id", nullable = false)
+    private EducationalProgram educationalProgram;
+
+    @Column(name = "repository_link")
+    @Nullable
+    private String repositoryLink;
+
+    @ManyToOne
+    @JoinColumn(name = "client_id", nullable = false)
+    private Client client;
+
+    @ManyToMany (cascade = {
+            CascadeType.PERSIST,
+            CascadeType.MERGE
+    })
+    @JoinTable(name = "project_student",
+            joinColumns = @JoinColumn(name = "project_id"),
+            inverseJoinColumns = @JoinColumn(name = "student_id")
+    )
+    private Set<Student> students;
+
+    @ManyToMany (cascade = {
+            CascadeType.PERSIST,
+            CascadeType.MERGE
+    })
+    @JoinTable(name = "project_clinic",
+            joinColumns = @JoinColumn(name = "project_id"),
+            inverseJoinColumns = @JoinColumn(name = "clinic_id")
+    )
+    private Set<Clinic> clinics;
+
+    @ManyToMany (cascade = {
+            CascadeType.PERSIST,
+            CascadeType.MERGE
+    })
+    @JoinTable(name = "project_tag",
+            joinColumns = @JoinColumn(name = "project_id"),
+            inverseJoinColumns = @JoinColumn(name = "tag_id")
+    )
+    private Set<Tag> tags;
+
+    @Column(name = "scientific_supervisor")
+    @Nullable
+    private String scientificSupervisor;
+
+    @Column(name = "result_link")
+    @Nullable
+    private String resultLink;
+
+    @OneToMany(mappedBy = "project")
+    private Set<Commit> commits;
+}
```

Comment:
Analysts also suggested adding a few points:
- time frame of the project
- project requirements
- requirements for performers
- work format